src/altinn.apps/apptemplates/aspnet/Altinn.App.PlatformServices/Implementation/InstanceClient.cs

```diff
@@ -111,7 +111,7 @@ public async Task<List<Instance>> GetInstances(Dictionary<string, StringValues>
             if (response.StatusCode == HttpStatusCode.OK)
             {
                 string instanceData = await response.Content.ReadAsStringAsync();
-                List<Instance> instances = JsonConvert.DeserializeObject<List<Instance>>(instanceData);
+                List<Instance> instances = JsonConvert.DeserializeObject<QueryResponse<Instance>>(instanceData).Instances;
```

Comment:
This bug is being fixed in a seperate PR. The file will de deleted as it is a duplicate with incorrect casing.

Reply:
It's been some back and forth here. Could you please push the content that you want?